client/src/test/kotlin/io/spine/examples/pingh/client/e2e/given/PersonalInteractionTestEnv.kt

```diff
@@ -66,3 +66,10 @@ internal fun List<MentionView>.updateStatusById(id: MentionId, status: MentionSt
             .setStatus(status)
             .vBuild()
     }
+
+/**
+ * Returns a random `MentionView` with `UNREAD` status from this list.
+ */
+internal fun List<MentionView>.randomUnread(): MentionView =
```

Comment:
What if there is no such mentions in this list? And we are returning a non-null object.